src/components/common/Select/index.tsx

```diff
@@ -0,0 +1,23 @@
+import * as S from "./styled";
+import type { SelectHTMLAttributes } from "react";
+
+interface Props extends SelectHTMLAttributes<HTMLSelectElement> {
+  options: { [key: string]: string };
+  placeholder?: string;
+}
+
+export default function Select({ options, placeholder, ...props }: Props) {
+  const optionKeys = Object.keys(options);
```

Comment:
```suggestion
interface Props extends SelectHTMLAttributes<HTMLSelectElement> {
  options: string[];
  placeholder?: string;
}

export default function Select({ options, placeholder, ...props }: Props) {
```

내부 목록들을 `options` 라는 object 로 받은 뒤 `Object.keys` 를 통해 key 의 배열로 변환해서 `.map` 으로 보여주고 계신데
처음 prop 으로 받을 때 부터 `string` 배열로 받는건 어떨까요?

Reply:
`option` 태그를 생성할 때, 실제 값(key)와 보여질 값(value)가 필요한데 `string[]` 2가지 정보를 어떻게 받을 수 있을까요??!

Reply:
아 제가 value 가 쓰인 곳을 못찾아서 말씀드린건데 잘못 봤네요
`options: {key: string, value: string}[]` 이런 배열도 가능은 하겠지만 큰 차이가 있는진 모르겠네요
이부분은 편하신 쪽으로 넘어가도 될 것 같습니다!